bin/reth/src/node/mod.rs

```diff
@@ -77,7 +76,7 @@ impl Command {
                 },
                 false,
             )
-            .push(SendersStage { batch_size: 1000, commit_threshold: 100 }, false);
+            .push(SendersStage { batch_size: 100, commit_threshold: 1000 }, false);
```

Comment:
how should we config this?

Reply:
`reth.toml` later on, for now we leave as is